build_scripts/tomcat_setup/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Download and extract Tomcat
+wget http://apache.mirrors.hoobly.com/tomcat/tomcat-8/v8.5.23/bin/apache-tomcat-8.5.23.tar.gz -P /tmp
```

Comment:
Can we add a flag to provide the tar file if you don't have an internet connection on the server?